packages/deployments/contracts/src/init/init.ts

```diff
@@ -53,6 +57,21 @@ export const sanitizeAndInit = async () => {
     throw new Error(`Network should be either testnet or mainnet, network: ${network}`);
   }
 
+  if (typeof liquidity !== "boolean") {
+    throw new Error(`"liquidity" option must be a boolean, liquidity: ${liquidity}`);
+  }
+
+  if (!messaging) {
+    throw new Error(
+      `"messaging" option must be true, messaging: ${messaging}. (can be false in the future, but this script should be updated)`,
+    );
+  }
+  // TODO: when messaging
```

Comment:
?

Reply:
honestly idk -- update when we want to init only liquidity layer is the guess?